tests/RobotFramework/tests/cumulocity/log/log_operation.robot

```diff
@@ -40,6 +41,34 @@ Manual log_upload operation request
     ...    payload={"status":"init","tedgeUrl":"http://127.0.0.1:8000/tedge/file-transfer/${DEVICE_SN}/log_upload/example-1234","type":"example","dateFrom":"${start_timestamp}","dateTo":"${end_timestamp}","searchText":"first","lines":10}
     ...    c8y_fragment=c8y_DownloadConfigFile
 
+Log operation uses correct tedgeUrl when file transfer service on diffent host
+    ${parent_ip}=    Get IP Address
+
+    ${CHILD_SN}=    Setup    skip_bootstrap=True
+    Set Device Context    ${CHILD_SN}
+    
+    # Set up a child device with only tedge-agent and connect both devices' MQTT broker and HTTP file transfer server
+    Execute Command    dpkg -i packages/tedge_*.deb packages/tedge-agent_*.deb
+    Execute Command    tedge config set http.bind.address 0.0.0.0
+    Execute Command    tedge config set mqtt.client.host ${parent_ip}
+    Restart Service    tedge-agent
+    ${child_ip}=       Get IP Address
+
+    Set Device Context    ${DEVICE_SN}
+    Execute Command    tedge config set mqtt.bind.address 0.0.0.0
+    Execute Command    tedge config set http.client.host ${child_ip}
+    Execute Command    tedge reconnect c8y
+    Stop Service       tedge-agent
+
+
+    ${start_timestamp}=    Get Current Date    UTC    -24 hours    result_format=%Y-%m-%dT%H:%M:%S+0000
+    ${end_timestamp}=    Get Current Date    UTC    +60 seconds    result_format=%Y-%m-%dT%H:%M:%S+0000
+    ${operation}=     Cumulocity.Create Operation
+    ...    description=Log file request
+    ...    fragments={"c8y_LogfileRequest":{"dateFrom":"${start_timestamp}","dateTo":"${end_timestamp}","logFile":"example","searchText":"first","maximumLines":10}}
+    Should Have MQTT Messages    te/device/main///cmd/log_upload/+    message_contains=tedgeUrl":"http://${child_ip}
+
+
```

Comment:
This test currently checks if correct FTS address is used in the payload, but will be changed in a next PR to check if the operation actually succeeds.

Reply:
I don't think we need to explicitly check the `tedgeUrl` address in the as this is just implementation detail. If the an invalid address is used then the operation will just fail anyway.

Reply:
For now I added the `tedgeUrl` check to show that the current code change had an effect, I'll remove `tedgeUrl` assert in the next PR.